src/backend/crossterm.rs

```diff
@@ -127,6 +119,194 @@ where
+    pub fn stdout() -> Self {
+        Self::new(std::io::stdout())
+    }
+}
+
+impl CrosstermBackend<std::io::Stderr> {
+    /// Creates a new `CrosstermBackend` with `std::io::stderr`.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// use ratatui::prelude::*;
+    ///
+    /// let backend = CrosstermBackend::stderr();
+    /// ```
+    pub fn stderr() -> Self {
+        Self::new(std::io::stderr())
+    }
+}
+
+impl<W: Write> CrosstermBackend<W> {
+    /// Converts the `CrosstermBackend` into a [`Terminal`] instance.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// use ratatui::backend::CrosstermBackend;
+    ///
+    /// let terminal = CrosstermBackend::stdout()
+    ///     .with_raw_mode()?
+    ///     .with_alternate_screen()?
+    ///     .with_mouse_support()?
+    ///     .into_terminal()?;
+    /// # std::io::Result::Ok(())
+    /// ```
+    pub fn into_terminal(self) -> io::Result<Terminal<Self>> {
+        Terminal::new(self)
+    }
+
+    /// Converts the `CrosstermBackend` into a [`Terminal`] instance with options.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// use ratatui::{backend::CrosstermBackend, TerminalOptions};
+    ///
+    /// let options = TerminalOptions {
+    ///     viewport: Viewport::Inline(10),
+    /// };
+    /// let terminal = CrosstermBackend::stdout().into_terminal_with_options(options)?;
+    /// # std::io::Result::Ok(())
+    /// ```
+    pub fn into_terminal_with_options(
+        self,
+        options: TerminalOptions,
+    ) -> io::Result<Terminal<Self>> {
+        Terminal::with_options(self, options)
+    }
+
+    /// Converts the `CrosstermBackend` into a [`Terminal`] instance with default settings.
+    ///
+    /// This enables raw mode and switches to the alternate screen. Mouse support is not enabled.
+    ///
+    /// Raw mode and alternate screen are restored when the `CrosstermBackend` is dropped.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// use ratatui::backend::CrosstermBackend;
+    ///
+    /// let terminal = CrosstermBackend::stdout().into_terminal_with_defaults()?;
+    /// # std::io::Result::Ok(())
+    /// ```
+    pub fn into_terminal_with_defaults(self) -> io::Result<Terminal<Self>> {
+        let backend = self.with_raw_mode()?.with_alternate_screen()?;
+        Terminal::new(backend)
+    }
+
+    /// Enables raw mode for the terminal.
+    ///
+    /// Returns an [`io::Result`] containing self so that it can be chained with other methods.
+    ///
+    /// Raw mode is restored when the `CrosstermBackend` is dropped.
+    ///
+    /// # Example
+    ///
+    /// ```rust,no_run
+    /// use ratatui::backend::CrosstermBackend;
+    ///
+    /// let backend = CrosstermBackend::stdout().with_raw_mode()?;
+    /// # std::io::Result::Ok(())
+    /// ```
+    pub fn with_raw_mode(mut self) -> io::Result<Self> {
+        self.restore_raw_mode_on_drop = true;
+        terminal::enable_raw_mode()?;
```

Comment:
Currently, there are `Terminal` (ratatui) and `terminal` (crossterm) close together and not confusing at all. Using an explicit `crossterm::` here clarifies this confusion. 

```suggestion
        crossterm::terminal::enable_raw_mode()?;
```

(There are more places like this)

Reply:
I'll actually move these to the imports to avoid this.